docs/api/miio.ihcooker.rst

```diff
@@ -0,0 +1,7 @@
+miio.ihcooker module
+====================
+
+.. automodule:: miio.ihcooker
```

Comment:
Either remove this file or fix the hierarchy.

Reply:
Removed it.

Reply:
It's still here :-)